ui/StatusQ/src/StatusQ/Components/StatusItemSelector.qml

```diff
@@ -1,20 +1,21 @@
 import QtQuick 2.14
 import QtQuick.Layouts 1.14
-import QtQuick.Controls 2.14 as QC
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
-import StatusQ.Controls 0.1
 import StatusQ.Core.Utils 0.1
 
 /*!
    \qmltype StatusItemSelector
-   \inherits Rectangle
+   \inherits StatusFlowSelector
    \inqmlmodule StatusQ.Components
    \since StatusQ.Components 0.1
-   \brief It allows to add items and display them as a tag item with an image and text. It also allows to store and display logical `and` / `or` operators into the list. Inherits \l{https://doc.qt.io/qt-6/qml-qtquick-rectangle.html}{Item}.
+   \brief It allows to add items and display them as a tag item with an image
```

Comment:
Since I guess your intention for the new airdrop components is create a new component based on `StatusFlowSelector` and not based on `StatusItemSelector`, maybe we can rename that to something more explicit to `tags` mode visualization. WDYT?

Reply:
I didn't invent better name bc we already have `StatusTagsSelector` which is a different thing :)